commit be18e13aac2dc4bb2b42bee6b6c48515970bdce9
Author: Davide Cavalca <dcavalca@fb.com>
Date:   Wed Jul 27 14:22:26 2016 -0700

    yummy:patchX:tests--skip-process-1-tests-if-systemd-not-is-running.patch

tests: skip process 1 tests if systemd not is running

    
    No point running tests against process 1 if systemd is not running as that
    process. This is a rework of an unpublished patch by @9muir.

diff --git a/src/test/test-process-util.c b/src/test/test-process-util.c
index 562ad4a..9ada46b 100644
--- a/src/test/test-process-util.c
+++ b/src/test/test-process-util.c
@@ -40,6 +40,7 @@
 #include "stdio-util.h"
 #include "string-util.h"
 #include "terminal-util.h"
+#include "test-helper.h"
 #include "util.h"
 #include "virt.h"
 
@@ -357,7 +358,7 @@ int main(int argc, char *argv[]) {
                 (void) parse_pid(argv[1], &pid);
                 test_get_process_comm(pid);
         } else {
-                test_get_process_comm(1);
+                TEST_REQ_RUNNING_SYSTEMD(test_get_process_comm(1));
                 test_get_process_comm(getpid());
         }
 
